runner/app/pipelines/utils/audio.py

```diff
@@ -19,7 +19,46 @@ class AudioConverter:
     """Converts audio files to different formats."""
 
     @staticmethod
-    def convert(input_bytes: bytes, output_extension: str, output_codec=None) -> bytes:
+    def to_ndarray(input_bytes: bytes) -> np.ndarray:
+        #inspired by https://github.com/SYSTRAN/faster-whisper/blob/d889345e071de21a83bdae60ba4b07110cfd0696/faster_whisper/audio.py
+        """Converts audio in media file to a NumPy array.
+        
+        Args:
+            input_bytes: The audio file as bytes to convert.
+
+        Returns:
+            The audio file as a NumPy array.
+        """
+        output_buffer = BytesIO()
+        input_buffer = BytesIO(input_bytes)
+        resampler = av.audio.resampler.AudioResampler(
+            format="s16",
+            layout="mono",
+            rate=16000,
+        )
+        
+        audio_array = None
+        try:
+            input_container = av.open(input_buffer, mode="r")
+            for stream in input_container.streams.audio:
+                for frame in input_container.decode(stream):
+                    resampled_frame = resampler.resample(frame)
+                    array = resampled_frame[0].to_ndarray()
+                    dtype = array.dtype
+                    output_buffer.write(array)
+            
+            audio_array = np.frombuffer(output_buffer.getbuffer(), dtype=dtype)
+            audio_array = audio_array.astype(np.float32) / 32768.0
+            
+        except Exception as e:
+            raise AudioConversionError(f"Error during audio conversion to numpy array: {e}")
+        finally:
+            input_container.close()
+
+        return audio_array
+    
+    @staticmethod
+    def convert(input_bytes: bytes, output_extension: str, output_codec: str) -> bytes:
         """Converts an audio file to a different format.
 
         Args:
```

Comment:
We can remove the convert method now, it's no longer used

Reply:
Not used right now but I felt was worth keeping since we are still building out the feature.  Having a way to request a different audio format in return may be worth keeping around.

Reply:
Sounds good to me